src/test/common/Throttle.cc

```diff
@@ -217,41 +218,36 @@ TEST_F(ThrottleTest, wait) {
 }
 
 TEST_F(ThrottleTest, destructor) {
-  Thread_get *t;
-  {
-    int64_t throttle_max = 10;
-    Throttle *throttle = new Throttle(g_ceph_context, "throttle", throttle_max);
+  int64_t throttle_max = 10;
+  Throttle *throttle = new Throttle(g_ceph_context, "throttle", throttle_max);
```

Comment:
i'd suggest use `unique_ptr` or `scoped_ptr`. otherwise static analyzer would complain, because if we bail out if any of the following `ASSERT_*()` fails, we won't `delete throttle`, hence memory leak.